File: estudo/neri/algoritmo/py/InssSalariolb.py
nome = input("\n Informe seu nome")
salario_bruto = float(input("\n Informe seu salario"))
inss = salario_bruto * 9 /100
salario_liquido = salario_bruto - inss
print("\n Nome: ",nome)
print(f"\n Seu salario bruto foi de: ",salario_bruto)
print(f"\n O valor do inss foi de: ",inss)
print(f"\n O seu salario liquido foi de: ",salario_liquido)

File: estudo/neri/algoritmo/c/InssSalariolb.c
#include <stdio.h>
int main(){
    float salario_bruto, inss, salario_liquido;
    char nome[50];
    printf("\n Informe o seu nome:");
    scanf("%s",nome);
    printf("\n Informe seu salario bruto");
    scanf("%f",&salario_bruto);
    inss = salario_bruto * 9 /100;
    salario_liquido = salario_bruto - inss;
    printf("\n Seu nome: %s",nome);
    printf("\n O valor do seu salario bruto foi de: %f",salario_bruto);
    printf("\n O valor descontado do inss foi de: %f",inss);
    printf("\n Seu salario liquido foi de: %f",salario_liquido);
}
